src/routes/categories/components/CategoryListItem.js

```diff
@@ -81,8 +81,9 @@ class CategoryListItem extends React.PureComponent<PropsType> {
 
   renderTitle (): React.Node {
     const { query, category, theme } = this.props
-    return <CategoryCaption searchWords={query ? [query] : []} aria-label={category.title} sanitize={normalizeSearchString}
-                            highlightStyle={{ backgroundColor: theme.colors.themeColor }}
+    return <CategoryCaption searchWords={query ? [query] : []} aria-label={category.title}
+                            sanitize={normalizeSearchString}
+                            highlightStyle={{ backgroundColor: theme.colors.backgroundColor, fontWeight: 'bold' }}
```

Comment:
`backgroundColor` Should be removed here. There is no reason to colorize the background of the text.
```suggestion
                            highlightStyle={{ fontWeight: 'bold' }}
```

Reply:
The highlighter has a default highlight color. If I remove the backgroundColor it will be yellow again.